apps/meteor/ee/app/livechat-enterprise/server/settings.ts

```diff
@@ -165,6 +165,48 @@ export const createSettings = async (): Promise<void> => {
 				modules: ['livechat-enterprise'],
 				invalidValue: '',
 			});
+
+			await this.add('Livechat_Block_Unknown_Contacts', false, {
```

Comment:
Aren't we missing the i18n keys for all the settings?

Reply:
They are not being rendered anywhere now, but I added them in the frontend feature branch, just cherry-picked the commit in here so we could have the values of this settings